### DISCUSSION TITLE
E-paper/e-ink UI

### DISCUSSION THREAD

**Discussion by slavalamp** [FossifyOrg/General-Discussion#133] [Ideas]
Many Android devices' (e-readers, tablets, phones) screens and some monitors use the electronic paper (aka e-paper or e-ink) technology, which means they have a limited amount of colors (often just 16 shades of gray, sometimes the contrast is also pretty poor) and a very low refresh rate.

I'd love it if Fossify apps had a UI option optimized for those screens: high contrast with only black and white colors and no animations (including scrolling, it should be actual pagination instead).

I'm interested in contributing to making it real as well, but I don't have any experience developing with Kotlin or for Android, so I have no idea where to start.

**Comment by naveensingh** [comment]
Closing for now: https://github.com/orgs/FossifyOrg/discussions/166#discussioncomment-9417721
